pkg/sidecar/sidecar.go

```diff
@@ -277,7 +211,7 @@ func (s *Sidecar) dumpBundles(svidResponse *proto.X509SVIDResponse) error {
 // writeCerts takes an array of certificates,
 // and encodes them as PEM blocks, writing them to file
 func (s *Sidecar) writeCerts(file string, certs []*x509.Certificate) error {
-	var pemData []byte
+	pemData := make([]byte, 0, len(certs))
```

Comment:
This change pre-allocates a number of bytes equal to the number of certs being written, which doesn't make sense.